day1/tests/coordinateCoordinatorTest.py

```diff
@@ -0,0 +1,28 @@
+import unittest
+from ..src.coordinateCalculator import CoordinateCalculator
+
+class TestDay1(unittest.TestCase):
+
+    def test_calcTaxiCabGeometryFromR2L3(self):
```

Comment:
Vc escreveu os testes da seguinte forma:
Definiu um cenário - tipo esse que é "R2, L3", ou lá em baixo que é "R2, R2, R2" - e depois testou seus 2 métodos - "total_distance" e "firstLocationVisitedTwiceDistance" - dentro de cada cenário.

Podemos até dizer que a sua app está testada e se vc refatorar algum método, a suíte de testes vai quebrar e avisar que algo de errado não está certo.

Mas geralmente a gente costuma testar 1 método de cada vez, ou seja, poderíamos fazer assim:
```python
def test_total_distance_using_coordinates_from_right_then_left(self):
    calculator = CoordinateCalculator("R2, L3")
    calculator.calcTaxiCabGeometryFrom()

    self.assertEqual(calculator.total_distance, 5)

def test_first_location_visited_twice_using_coordinates_from_right_then_left(self):
    calculator = CoordinateCalculator("R2, L3")
    calculator.calcTaxiCabGeometryFrom()

    self.assertEqual(calculator.firstLocationVisitedTwiceDistance, 0)
```

E aí cada 1 desses testes que vc fez, vai virar 2.

Aí vc poderia pensar:
_ Poxa, mas tá repetindo o `setup` do teste nesses 2 métodos, como fica o DRY?

Mas para escrever testes a gente geralmente não pensa muito no DRY (Don't Repeat Yourself)! Mas sim no DAMP (Descriptive And Meaningful Phrases)!

Ou seja, não me importa se eu estiver repetindo código aqui nos testes, mas me importa que os testes estejam legíveis e fáceis de entender!

Em uma googlada rápida achei isso sobre: https://stackoverflow.com/a/11837973/4842544

(PS: O que estou dizendo é do que eu sei e eu sou Dev Ruby, pode ser que a comunidade Python não siga essas mesmas práticas que estou dizendo, mas pra mim elas fazem total sentido independente da linguagem que vc está usando).